rest_api/workloads_apis/buildconfig-build-openshift-io-v1.adoc

```diff
@@ -1200,7 +1200,9 @@ Type::
 
 | `lastTriggeredImageID`
 | `string`
-| lastTriggeredImageID is used internally by the ImageChangeController to save last used image ID for build
+| lastTriggeredImageID is used internally by the ImageChangeController to save last used image ID for build. Deprecated. {product-title} version 4.9 will remove support for `lastTriggeredImageID` and ignore it. Then, image change triggers will not start a build based on changes to the `lastTriggeredImageID` field in the `BuildConfig` spec. Instead, the image IDs that trigger a build will be recorded in the status of the `BuildConfig` object, which most users cannot change.
+
+Therefore, update scripts and jobs that inspect `checkingbuildConfig.spec.triggers[i].imageChange.lastTriggeredImageID` accordingly. (link:https://issues.redhat.com/browse/BUILD-213[*BUILD-213*])
```

Comment:
@jboxman I'm noticing some `..` formatting throughout this file, like you see on line 1210 below, and wondering if these should be H2 or H<n> titles instead. WDYT?

Reply:
formatting aside @rolfedh the text for the spec field looks good here.